app/lib/common/providers/space_providers.dart

```diff
@@ -183,6 +183,32 @@ final briefSpaceItemsProviderWithMembership =
   return items;
 });
 
+final spaceSearchValueProvider = StateProvider<String?>((ref) => null);
```

Comment:
would want to keep it ```.autoDispose``` so it flushes out when not listened?

Reply:
I have taken reference from `chatSearchValueProvider`[ (chat_providers.dart)](https://github.com/acterglobal/a3/blob/main/app/lib/features/chat/providers/chat_providers.dart).

But ya you are right. It should be` .autoDispose`. I will do the required changes here.